runtime/vm/vmthread.cpp

```diff
@@ -755,7 +755,7 @@ threadParseArguments(J9JavaVM *vm, char *optArg)
 #else /* defined(J9VM_ARCH_X86) */
 	**(UDATA **)omrthread_global((char *)"parkSleepCpuUtilThreshold") = 100;
 #endif /* defined(J9VM_ARCH_POWER) */
-	vm->cpuUtilCacheInterval = 5000000000;
+	vm->cpuUtilCacheInterval = 5;
```

Comment:
Please add a comment in `j9nonbuilder.h` noting that the `cpuUtilCacheInterval` is a number of seconds.